css-grid-classes/styles.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.column {
```

Comment:
this is good for the `column-one-fourth` class, make sure to create classes for the rest of them.